target-platform/dev.target

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?>
+<target name="dev">
```

Comment:
Maybe a more expressive name like `m2e-with-running-eclipse` would be better.
Some devs have multiple projects in their workspace.